helm-chart/binderhub/templates/configmap.yaml

```diff
@@ -3,19 +3,22 @@ apiVersion: v1
 metadata:
   name: binder-config
 data:
-{{- /* Important: only pick relevant subset of values. Including too much here can trigger unnecessary binderhub pod restarts */}}
-{{- $values := pick .Values "config" "cors" "dind" "extraConfig" }}
-{{- /* make a copy to avoid modifying the original */ -}}
-{{- $_ := set $values "config" (merge dict .Values.config) }}
-{{- $_ := set $values.config "BinderHub" (merge dict .Values.config.BinderHub) }}
-{{- /* trim secret values. Update here if new secrets are added! */ -}}
-{{- /* every 'omit' here should be matched with a corresponding 'pick' in secret.yaml */ -}}
-{{- if $values.config.GitHubRepoProvider }}
-{{- $_ := set $values.config "GitHubRepoProvider" (omit .Values.config.GitHubRepoProvider "client_id" "client_secret" "access_token") }}
-{{- end }}
-{{- if $values.config.GitLabRepoProvider }}
-{{- $_ := set $values.config "GitLabRepoProvider" (omit .Values.config.GitLabRepoProvider "private_token" "access_token") }}
-{{- end }}
+  # FIXME: Stop splitting up parts into configmap/secret as it adds complexity,
+  #        instead only use a secret. For the equivalent PR in z2jh, see:
+  #        https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/1682
```

Comment:
Did you mean for this comment to be included in the actual configmap, or as a template comment?

Reply:
I didn't mind if it was included in the configmap, but as the checksum is changed by doing this, I figure should make it a helm template to avoid triggering a restart of the binder-pod for no reason.